src/client/src/orangehrmAdminPlugin/pages/languageTranslation/languageTranslation.vue

```diff
@@ -19,7 +19,12 @@
  -->
 
 <template>
-  <div class="orangehrm-background-container">
+  <oxd-alert
```

Comment:
@ManulMax what's this requirement? we normally show no records found using a toast message right?

Reply:
Changed it